smslibrary/src/main/java/com/eis/smslibrary/DefaultSMSParseStrategy.java

```diff
@@ -0,0 +1,35 @@
+package com.eis.smslibrary;
+
+import androidx.annotation.NonNull;
+
```

Comment:
A class description would fit well, like "Default parsing strategy used in {@link SMSMessageParser}, uses hidden character etc..."